modules/hcp-ibmz-adding-reg-ca-hostedcluster.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-adding-registry-ca-hostedcluster_{context}"]
+= Adding the Registry Certificate Authority to the Hosted Cluster
+
+When you are creating {hcp}, include the `additional-trust-bundle` and `image-content-sources`. This will allow the hosted cluster to inject the Certificate Authority into the Data plane workers so that the images are pulled from the registry.
```

Comment:
```suggestion
When you are deploying {hcp} on {ibm-z-title} in a disconnected environment, include the `additional-trust-bundle` and `image-content-sources` resources. Those resources allow the hosted cluster to inject the certificate authority into the data plane workers so that the images are pulled from the registry.
```
Check my revision for accuracy. I assumed that `additional-trust-bundle` and `image-content-sources` are resources.